packages/wonder-blocks-link/components/link.js

```diff
@@ -20,12 +20,6 @@ export type SharedProps = {|
      */
     href: string,
 
-    /**
-     * Whether to display a caret after the text.
-     */
-    // TODO(yejia): Add once we have an Icon component
-    caret: boolean,
```

Comment:
I'm confused - didn't you just add this in an other PR?

Reply:
I thought this prop was missing because of the `TODO`.  It turns out `caret` isn't actually part of the spec.